src/coreclr/jit/codegen.h

```diff
@@ -274,7 +274,7 @@ class CodeGen final : public CodeGenInterface
     void genEnregisterOSRArgsAndLocals();
 #endif
 
-    void genHomeSwiftStructParameters(bool handleStack);
+    void genHomeSplitStructParameters(bool handleStack);
```

Comment:
"split" is a bit misleading for Swift, these parameters are not necessarily "split" more than any other parameter passed across multiple registers would be. Maybe we can name it something like `genHomeReassembledParameters`?

Reply:
Maybe we should just name it `genHomeStackParams` to match `genHomeRegisterParams` (anticipating the removal of the argument)